daisi/src/cpps/logical/algorithms/disposition/iterated_auction_disposition_initiator.cpp

```diff
@@ -20,7 +20,7 @@
 #include "ns3/simulator.h"
 
 using namespace daisi::material_flow;
-using namespace daisi::cpps::mrta::model;
+using namespace daisi::cpps::amr;
```

Comment:
The [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Namespaces) discourages using-directives. Is this necessary here, e.g., because it enhances the readability by a good amount?